Doc/reference/import.rst

```diff
@@ -922,6 +922,53 @@ it is sufficient to raise :exc:`ModuleNotFoundError` directly from
 while raising an exception terminates it immediately.
 
 
+Package Relative Imports
+========================
+
+Relative imports use leading dots. A single leading dot indicates a relative
+import, starting with the current package. Two or more leading dots indicate a
+relative import to the parent(s) of the current package, one level per dot
+after the first. For example, given the following package layout::
+
+    package/
+        __init__.py
+        subpackage1/
+            __init__.py
+            moduleX.py
+            moduleY.py
+        subpackage2/
+            __init__.py
+            moduleZ.py
+        moduleA.py
+
+Given that the current file is either ``moduleX.py`` or ``subpackage1/__init__.py``,
+the following is the syntax for relative imports:
```

Comment:
I think this can be reworded as "the following relative imports are valid:" or "the following are valid relative imports:". The first part of the sentence sets up a sort of conditional, but all of these are examples of valid relative import *syntax* even if the import itself would fail.

One other thing to note, I think if you use `::` instead of `:` at the end here, it will make the code after it render as a code block *and* have a colon at the end of the lead-in sentence (I usually mimic what I've seen other people doing here, I actually never looked up what this syntax actually does 😉).